run_deepseek_batch_split.sh

```
@@ -0,0 +1,398 @@
+
+multi_round=1
+disable_overlap_scheduler="false"
+
+# Default batch split configuration
+USE_BATCH_SPLIT_MODEL="true"
+BATCH_SPLIT_RATIO=0.5
+USE_SEPARATE_STREAMS="true"
+SYNC_AFTER_ATTENTION="true"
+ENABLE_LAYER_COMPLETION_EVENTS="false"
+ENABLE_INTRA_LAYER_BATCH_SPLITTING="true"
+NABLE_ATTENTION_METADATA_SPLITTING="true"
```

Shellcheck (0.10.0)

[warning] NABLE_ATTENTION_METADATA_SPLITTING appears unused. Verify use (or export if used externally).

(SC2034)